### ISSUE TITLE
Indented list not correctly converted

### ISSUE THREAD

**Issue by gutobenn** [thephpleague/html-to-markdown#19]
The list
````
<ul>
	<li>normal list item
		<ul>
			<li>indented item</li>
			<li>another one</li>
		</ul>
	</li>
	<li>normal item again</li>
</ul>
````

should be converted to:

````
- normal list item
 - indented item
 - another one
- normal item again
````

But it is being converted to:

````
- normal list item- indented item
- another one
- normal item again
````

**Comment by colinodell** [comment]
It looks like the current version doesn't track the current list level, so this will likely require a partial/full rewrite before we can get it working properly.

**Comment by marijnvdwerf** [comment]
Would this change to `ListItemConverter` solve those issues?

```php
$value = implode("\n", $lines);

if ($list_type === 'ul') {
    $indent = '- ';
} else {
    $number = $element->getSiblingPosition();
    $indent = $number . '. ';
}

$value = trim($value);
$value = preg_replace('~\n~', "\n" . str_repeat(' ', strlen($indent)), $value);

$markdown = $indent . $value . "\n";
```

**Comment by colinodell** [comment]
Fixed by #84
